source/aggregation-pipeline-builder.txt

```diff
@@ -112,35 +112,31 @@ your chosen view mode:
       .. procedure:: 
```

Comment:
[question, non-blocking] Does it make sense to change the name of this page to "Build an Aggregation Pipeline"?

Reply:
sgtm!